matterbridge.go

```diff
@@ -10,53 +10,76 @@ import (
 	"github.com/42wim/matterbridge/gateway"
 	"github.com/42wim/matterbridge/gateway/bridgemap"
 	"github.com/google/gops/agent"
-	prefixed "github.com/matterbridge/logrus-prefixed-formatter"
+	"github.com/matterbridge/logrus-prefixed-formatter"
```

Comment:
```suggestion
	prefixed "github.com/matterbridge/logrus-prefixed-formatter"
```
because the path isn't the same as the package name, makes it better visible (unless you have strong arguments to not do this)

Reply:
Yeah my linter was actually complaining about it. Have reinstated it.